edb/server/compiler/compiler.py

```diff
@@ -520,7 +520,7 @@ def _compile_and_apply_ddl_command(self, ctx: CompileContext, cmd):
 
         if debug.flags.delta_execute:
             debug.header('Delta Script')
-            debug.dump_code('\n'.join(sql), lexer='sql')
+            debug.dump_code(b'\n'.join(sql), lexer='sql')
```

Comment:
Don't you need to decode it? Or it works with bytes too?

Reply:
I've added a `b` (as Elivs suggested) and run bootstrap. The bug fixed. I'm not sure how all of that works actually.